website/js/ladder.js

```diff
@@ -26,7 +26,23 @@ var LadderPanel = Panels.StaticPanel.extend({
 	events: {
 		'change select[name=standing]': 'changeStanding',
 		'click button[name=openReset]': 'openReset',
-		'click button[name=cancelReset]': 'cancelReset'
+		'click button[name=cancelReset]': 'cancelReset',
+		'click button[name=copyUrl]': 'copyUrl',
+	},
+	copyUrl: function (e) {
+		var token = e.currentTarget.value;
+		// this works since this is in the root as well
+		var url = 'https://' + document.location.hostname + '/resetpassword/' + token;
+		var dummy = document.createElement("input");
+		var placeholder = $('.pfx-topbar')[0];
+		dummy.id = "dummy";
+		dummy.value = url;
+		dummy.style.position = 'absolute';
```

Comment:
does `display:none` not work here?

Reply:
Probably, i just stole this from the other place we do copying that's not in the main site.